packages/tm-themes/index.d.ts

```diff
@@ -1,10 +1,27 @@
+/** https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository */
+export interface FundingInfo {
```

Comment:
I didn't see any shared dependency for these packages, so I had to copy/paste the type definition. Let me know if there's a preferred alternative.